src/sentry/hybridcloud/models/cacheversion.py

```diff
@@ -19,7 +19,7 @@ def incr_version(cls, key: str) -> int:
             cls.objects.create_or_update(
                 key=key, defaults=dict(version=1), values=dict(version=F("version") + 1)
             )
-            return cls.objects.filter(key=key).values("version").first()["version"]
+            return cls.objects.get(key=key).version
```

Comment:
This is a little different now right, since it's turned into a `select *`

Reply:
I suspect it isn't because the only fields are (key, version)

Reply:
sgtm